src/Instagram/Auth/Checkpoint/ImapClient.php

```diff
@@ -101,6 +101,9 @@ private function available(): void
     public function getLastInstagramEmailContent(int $try = 1): string
     {
         $resource  = imap_open('{' . $this->getServer() . '/' . $this->getConnectionType() . '/ssl}INBOX', $this->getLogin(), $this->getPassword());
```

Comment:
Should we add `@` to supress the warning `Warning: imap_open(): Couldn't open stream {imap.gmail.com:993/imap/ssl}INBOX in Instagram\Auth\Checkpoint\ImapClient->getLastInstagramEmailContent() (line 103 of src/Instagram/Auth/Checkpoint/ImapClient.php)` as well?

Reply:
For sure. We could add this package to to suppress error (https://github.com/thecodingmachine/safe) but, I think it will be overkill.